docker_daemon/check.py

```diff
@@ -589,18 +586,17 @@ def _report_container_count(self, containers_by_id):
 
         for state in per_state_count:
             if state:
-                m_func(self, 'docker.container.count', per_state_count[state], tags=['container_state:%s' % state.lower()])
+                self.gauge('docker.container.count', per_state_count[state], tags=['container_state:%s' % state.lower()])
```

Comment:
Have you checked the impact of this change? Does it break the default dashboard? Can we still get the container count per host for example? I'm worried this could make some levels of aggregation impossible.